cling/staticinit/ROOT-10426/CMakeLists.txt

```diff
@@ -10,11 +10,7 @@ endif()
 
 add_library(ROOT10426 SHARED
    ROOT10426.cxx)
-if(MSVC AND DEFINED ROOT_SOURCE_DIR)
-  target_link_libraries(ROOT10426 PUBLIC Core)
-else()
-  target_link_libraries(ROOT10426 PUBLIC ROOT::Core)
-endif()
+target_link_libraries(ROOT10426 PUBLIC ROOT::Core)
```

Comment:
Nice! What changed that we can now do this on Windows too?

Reply:
I think it was mostly due to something missing between the builtin and standalone builds (hence the `AND DEFINED ROOT_SOURCE_DIR`), which is now fixed